Jenkinsfile

```diff
@@ -280,11 +280,15 @@ def generateStep(version){
         withEnv(["GO_VERSION=${version}"]) {
           // Another retry in case there are any environmental issues
           // See https://issuetracker.google.com/issues/146072599 for more context
+          deleteDir()
+          unstash 'source'
```

Comment:
IIRC `before_install` will fail if you run it twice and it already fetched some stuff. That might be why the `deleteDir` is inside the retry block.

Reply:
Mmm, I see, let me move it back to the `before_install` step. Related to the test failures, which also reproduces in master:

```log
[2020-08-05T09:19:25.302Z] + make install precheck check-modules

[2020-08-05T09:19:25.562Z] go get -v -t ./...

[2020-08-05T09:19:26.542Z] google.golang.org/grpc/internal/credentials

[2020-08-05T09:19:26.542Z] # google.golang.org/grpc/internal/credentials

[2020-08-05T09:19:26.542Z] ../../google.golang.org/grpc/internal/credentials/spiffe.go:34:70: state.PeerCertificates[0].URIs undefined (type *x509.Certificate has no field or method URIs)

[2020-08-05T09:19:26.542Z] ../../google.golang.org/grpc/internal/credentials/spiffe.go:38:47: state.PeerCertificates[0].URIs undefined (type *x509.Certificate has no field or method URIs)

[2020-08-05T09:19:26.543Z] ../../google.golang.org/grpc/internal/credentials/spiffe.go:56:35: state.PeerCertificates[0].URIs undefined (type *x509.Certificate has no field or method URIs)

[2020-08-05T09:19:26.543Z] Makefile:44: recipe for target 'install' failed

[2020-08-05T09:19:26.543Z] make: *** [install] Error 2

script returned exit code 2
```